src/main/java/bio/terra/cli/command/cromwell/GenerateConfig.java

```diff
@@ -22,6 +23,8 @@ public class GenerateConfig extends BaseCommand {
 
   @Override
   protected void execute() {
+    String absolutePath =
+        new File("src/main/resources/configs/generate-config.sh").getAbsolutePath();
```

Comment:
I found if I change the path from `src/main/resources/configs/generate-config.sh` to `generate-config.sh`. It also works to generate the config in the correct place... Not sure whether I should specify it like this. @yuhuyoyo

Reply:
really? do you know why is that?

Reply:
Not really sure the reason. But I checked the example for getAbsolutePath(). it should be relative path based on current file. https://www.baeldung.com/java-path#2-getabsolutepath

Reply:
right but doesn't make sense to me the relative path can find generate-config.sh if you are not in that directory. 

i think we should use getClassloader(). it has proven to work in the existing code path.  https://stackoverflow.com/a/17351116